kahuna/public/js/preview/image.html

```diff
@@ -61,7 +61,7 @@
                   title="this image has crops"
                   ng:if="image | hasExportsOfType:'crop'">
 
-                <i class="material-icons material-icons--small">crop</i>
+                <gr-icon small="true">crop</gr-icon>
```

Comment:
couldn't work out a way to use html5 attributes, i.e. `<gr-icon small>crop</gr-icon>`